diff -ur pxp-1.1.4/src/pxp-wlex/pxp_wlex.mlw pxp-1.1.4-/src/pxp-wlex/pxp_wlex.mlw
--- pxp-1.1.4/src/pxp-wlex/pxp_wlex.mlw	Fri Mar 15 23:16:45 2002
+++ pxp-1.1.4-/src/pxp-wlex/pxp_wlex.mlw	Sun May 12 00:10:10 2002
@@ -28,6 +28,8 @@
 
   let dummy_entity = ( new dummy_entity : entity_id )
 
+  let sub_lexeme lb st ln = String.sub (Lexing.lexeme lb) st ln
+
   (* The following tokens are pre-allocated to reduce the load on the
    * GC.
    *)
